packages/react-data-grid-addons/src/draggable/RowDragLayer.js

```diff
@@ -69,9 +69,9 @@ class CustomDragLayer extends Component {
           if (c.formatter) {
             const Formatter = c.formatter;
             const dependentValues = typeof c.getRowMetaData === 'function' ? c.getRowMetaData(item, c) : {};
-            cells.push(<td key={`dragged-cell-${rowIdx}-${c.key}`} className="react-grid-Cell" style={{ padding: '5px' }}><Formatter dependentValues={dependentValues} value={item[c.key]} /></td>);
+            cells.push(<td key={`dragged-cell-${rowIdx}-${c.key}`} className="rdg-cell" style={{ padding: '5px' }}><Formatter dependentValues={dependentValues} value={item[c.key]} /></td>);
```

Comment:
`{ padding: 5 }`? Any ideas why this is not moved to css?

Reply:
This PR doesn't aim to fix every styling issues in the repo, it's just trying to address issues that arose after removing the bootstrap styles. There are too many things to fix or cleanup otherwise.